ci/docker/docker-compose.yml

```diff
@@ -4,7 +4,8 @@ services:
     php-fpm.stepup.example.com:
         build:
             context: ../
-            dockerfile: docker/php-fpm/Dockerfile-php72
+#            # override this in a dedicated dockerfile per php version
+#            dockerfile: docker/php-fpm/Dockerfile-php72
```

Comment:
Maybe comment this out entirely?

Reply:
I've deliberately let it there so you know exactly what you need to change to let it work.